lib/tasks/recovery.rake

```diff
@@ -0,0 +1,6 @@
+namespace :recovery do
+  desc "Requeue content changes and messages over one-hour old for which jobs have been lost"
+  task recover_lost_jobs: :environment do
+    Recovery::RecoverLostJobs.run
```

Comment:
This should be a worker, that runs periodically: we're making this change so that no manual intervention is required when something is lost.